appeals/web/src/server/lib/mappers/appeal.mapper.js

```diff
@@ -1564,3 +1582,11 @@ export function generateDecisionDocumentDownloadHtml(appealDetails, linkText = '
 
 	return html;
 }
+
+/**
+ * @param {string | undefined} lpaQStatus
+ * @returns {boolean}
+ */
+function checkWhetherToDisplayChangeLinksBasedOnLPAQStatus(lpaQStatus) {
```

Comment:
Minor one - I'm generally in favour of long, descriptive symbol names, but this does feel a little bit verbose - would something like `shouldDisplayChangeLinksForLPAQStatus` be better?

Reply:
done